### PR TITLE
[Content] Add some more human news referring to Korath attacks in the core

### PR DESCRIPTION
**Pull request by roadrunner56**
**Content (Artwork / Missions / Jobs)**

## Summary
Added a few more news lines from merchant captains describing Korath raids in the core, as well as speculation as to what these unknown ships might be. Not sure if adding more of these makes the rate of these news items too common. If so, I can separate these into their own segment and make them a bit rarer.

## Testing Done
0

## Save File
nah

### CODE REVIEW

**Comment by Zitchas** [comment]
Overall, I like these. Good flavor, good lore, and in done cases, anti-lore. I would prefer to drop the last one, though. I don't think we need to increase the number of accurate statements about something that is supposed to be pretty well controlled and kept secret.

**Comment by roadrunner56** [comment]
Hmm, I was trying to go for the 'nutso conspiracy theorist that happens to be sort of correct, but for the wrong reasons'-type with the last news line. I'll see if I can edit it to be more wrong and less reliable-sounding in order to make it more ambiguous.

**Review by bene-dictator — APPROVED** [review]

**Comment by Zitchas** [comment]
The first change (replacing "every ship" with "every pirate, merchant, and syndicate ship") makes the sentence a bit clunky and loses the impact. Probably better the way it was.

The second change (making the kind of correct one a bit more wrong while still being vicinity-of-being-right) I think definitely improved it.

**Review by Zitchas — APPROVED** [review]
Looks good to me, although w may need a bunch of non Korath hinting news items to dilute their frequency a bit after this....

File: data/human/news.txt [hunk 1]
```diff
@@ -116,6 +116,11 @@ news "merchant in human space"
...
 			"The Syndicate tries to keep alien attacks under wraps, but if you've been around as long as I have, you'll see them eventually."
 			"I was working a Syndicate defense job once, expecting to only deal with some pirates. Instead, I had to drive off an alien warship that jumped in same way the Quarg do! They made us sign a non-disclosure agreement, so don't tell anyone I said that."
+			"I was doing some mining in the Core, when this massive ship came in out of nowhere and started attacking everything in sight! Must be some sort of new Syndicate tech they're testing."
+			"Don't believe the nonsense some captains spew about aliens in the Core. I guarantee you those ships are just some fancy new Syndicate tech."
+			"I don't think the Quarg are as peaceful as they say they are. Just last week I saw one of those jump-capable ships launch into some Core system and slaughter every other ship in sight!"
+			"The Syndicate must be getting pretty desperate with its pirate problem. I saw them unleash this massive attack ship in a pirate system. Probably some new defense initiatve."
```
Comment (ziproot) [inline data/human/news.txt:122]:
```suggestion
			"The Syndicate must be getting pretty desperate with its pirate problem. I saw them unleash this massive attack ship in a pirate system. Probably some new defense initiative."
```

**Review by ziproot — APPROVED** [review]

**Comment by warp-core** [comment]
The frequencies of these sorts of news lines are probably fine, for now.
They might be fairly common within this particular news item, but there are also a large number of news items to choose from before a message from this phrase is even considered.
If they do prove to be too common, though, I would suggest moving these lines into a phrase and embedding that phrase in a single option within this news block.

**Pull request merged**